README.md

```diff
@@ -1,34 +1,72 @@
+# BRL-CAD Web 2022
+
+[![IRC](https://img.shields.io/badge/chat-on%20irc%20channel-green.svg)](http://webchat.freenode.net/?channels=#brlcad)
+[![IRC](https://img.shields.io/badge/join-mailing%20list-green.svg)](https://lists.sourceforge.net/lists/listinfo/brlcad-devel)
+
+This is the official website of BRL-CAD community, where users can access some details about this community. User can view the history and details bout the BRL-CAD project and images using gallery.
+
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+## Table of Contents
+
+- [Installation](#installation)
+- [Setup and Run](#setup-and-run)
+- [Lint](#lint)
+- [Build and Deploy](#build-and-deploy)
+<!-- - [Contribution](#contribution) -->
+- [Learn More](#learn-more)
 
-First, run the development server:
+## Installation
+
+Clone this repo
+
+```sh
+git clone https://github.com/divyanshugarg36/brl-cad-web.git
+```
 
-```bash
+###### Install Nodejs :
+```
+sudo apt-get install nodejs
```

Comment:
Can you add prerequisites for the project as well?

Reply:
Added